crates/host/src/wasmbus/mod.rs

```diff
@@ -2355,6 +2403,42 @@ impl Host {
         }
     }
 
+    async fn store_actor_claims(&self, claims: jwt::Claims<jwt::Actor>) -> anyhow::Result<()> {
+        self.store_claims(
+            claims
+                .try_into()
+                .context("failed to convert actor claims")?,
+        )
+        .await
+    }
+
+    async fn store_provider_claims(
+        &self,
+        claims: jwt::Claims<jwt::CapabilityProvider>,
+    ) -> anyhow::Result<()> {
+        self.store_claims(
+            claims
+                .try_into()
+                .context("failed to convert provider claims")?,
+        )
+        .await
+    }
+
+    async fn store_claims(&self, stored_claims: StoredClaims) -> anyhow::Result<()> {
+        let key = format!("CLAIMS_{}", stored_claims.subject);
+
+        let bytes = serde_json::to_vec(&stored_claims)
+            .map_err(anyhow::Error::from)
+            .context("failed to serialize claims")?
+            .into();
+        self.data
+            .put(key, bytes)
+            .await
+            .map(|_| ())
```

Comment:
I think it would be much clearer to just remove this one and add `Ok(())` at the end

Reply:
I'm a fan of the toilet closure `|_| ()` 😉 IMO it's more clear that we're returning the result from `put` either way: either the error, or a mapped version of the ok. But I don't feel strongly about it, so I'll change it